jsonschema/tests/test_cli.py

```diff
@@ -66,36 +66,89 @@ def test_find_validator_in_jsonschema(self):
         )
         self.assertIs(arguments["validator"], Draft4Validator)
 
+    def test_none_validator(self):
+        arguments = cli.parse_args(
+            [
+                "--instance", self.instance_file,
+                self.schema_file,
+            ]
+        )
+        self.assertIs(arguments["validator"], LatestValidator)
+
+    def test_none_instance(self):
+        arguments = cli.parse_args(
+            [
+                self.schema_file,
+            ]
+        )
+        self.assertTrue(
```

Comment:
Any reason not to just make this `self.assertEqual(arguments["instances"], [])`?

Reply:
At first I thought it would be more precise to check the type explicitly, but it is true that `equal to []` also checks the type. I was afraid of Python’s weird side effects. But you’re right.